stretch_core/stretch_core/joint_trajectory_server.py

```diff
@@ -101,6 +102,71 @@ def merge_arm_joints(trajectory):
     return new_trajectory
 
 
+def preprocess_gripper_trajectory(trajectory):
```

Comment:
I find this function pretty intuitive and nicely explained in the comments. Thanks!